### PR TITLE
https://github.com/strongloop/loopback-next/issues/5595

### PR DESCRIPTION
**Pull request by HrithikMittal**
Fixes https://github.com/strongloop/loopback-next/issues/5595

## Checklist

- [ ] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Review by jannyHou — COMMENTED** [review]
@HrithikMittal Thanks for the docs. Please sign off your commit according to https://loopback.io/doc/en/contrib/doc-contrib.html#signing-off-commits-using-dco.

File: docs/site/Community-examples.md [hunk 1]
```diff
@@ -64,6 +64,24 @@ lists the examples created by the community.
...
+  <thead>
+    <tr>
+      <th width="160">Example</th>
+      <th width="280">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://github.com/HrithikMittal/loopback4-authorization">loopback4-Authorization</a></td>
+      <td>Implementation of your own authorization strategy in Loopback-4</td>
```
Comment (jannyHou) [inline docs/site/Community-examples.md:79]: Suggestion: maybe provide more description about how the authorization is realized. E.g. Implementation of your own authorization strategy in Loopback-4 by adding permission properties in User model.

**Comment by HrithikMittal** [comment]
> @HrithikMittal Thanks for the docs. Please sign off your commit according to [loopback.io/doc/en/contrib/doc-contrib.html#signing-off-commits-using-dco](https://loopback.io/doc/en/contrib/doc-contrib.html#signing-off-commits-using-dco).

Sorry, I can't do it. Please can you do it for me.

**Comment by dhmlau** [comment]
@HrithikMittal, we can fix the commit message linter issue, but need you to sign the DCO otherwise we cannot accept your PR. 
To sign, you can run:
```
git commit --amend -s
```

Hope it helps. Feel free to let us know or ping me on [Slack](https://join.slack.com/t/loopbackio/shared_invite/zt-8lbow73r-SKAKz61Vdao~_rGf91pcsw) if you have issue about signing. Thanks.

**Comment by raymondfeng** [comment]
@HrithikMittal I fixed the commit message and add `Signed-off-by: Adhikansh Mittal <adhikanshmittal@gmail.com>` for you. Please acknowledge that you confirm it by adding a comment here.

**Comment by HrithikMittal** [comment]
Yes @raymondfeng I confirm it.

**Comment by HrithikMittal** [comment]
and Thank you so much @raymondfeng

**Pull request merged**